capreolus/extractor/bertpassage.py

```diff
@@ -222,6 +207,30 @@ def parse_label_tensor(x):
         label = tf.map_fn(parse_label_tensor, parsed_example["label"], dtype=tf.float32)
 
         return (pos_bert_input, pos_mask, pos_seg, neg_bert_input, neg_mask, neg_seg), label
+    
+    def _filter_inputs(self, bert_inputs, bert_masks, bert_segs, n_valid_psg):
```

Comment:
Explicitly for training, this function randomly select one passage from the `n-passages`, this is done in extractor now so that `pytorch` and `tensorflow` trainer can both use it.